portal-frontend/src/app/features/edit-application/parcel-owners/parcel-owners.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ApplicationOwnerDto } from '../../../services/application-owner/application-owner.dto';
+import { ApplicationOwnerService } from '../../../services/application-owner/application-owner.service';
+import { ConfirmationDialogService } from '../../../shared/confirmation-dialog/confirmation-dialog.service';
+import { ApplicationOwnerDialogComponent } from '../application-owner-dialog/application-owner-dialog.component';
+
+@Component({
+  selector: 'app-parcel-owners[owners]',
+  templateUrl: './parcel-owners.component.html',
+  styleUrls: ['./parcel-owners.component.scss'],
+})
+export class ParcelOwnersComponent {
+  @Output() onAppUpdated = new EventEmitter<void>();
+
+  @Input()
+  public set owners(owners: ApplicationOwnerDto[]) {
+    this._owners = owners.sort((a, b) => {
+      if (a.displayName < b.displayName) {
```

Comment:
I remember I've seen this code somewhere above. May be a good candidate for helper function

Reply:
Done, moved to service and re-used